src/stylesheets/buttons.scss

```diff
@@ -0,0 +1,36 @@
+@import "colors";
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+.button {
+  outline-style: none;
+  background-color: $tc-white;
+  border: 1px solid $silver-gray;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-style: normal;
+  font-size: 14px;
```

Comment:
Are we making the button fixed size? If we use rem/em, it can scale based on the user's browser font size.

Reply:
I think I'm only fixing the height of the button container? The width should scale itself by the text content. But using em for height is probably good idea 👍

Reply:
Made the height: 40px to height: 2.5em.